### PR TITLE
fix namespace gaps for mandiff.

### PR DESCRIPTION
**Pull request by morvencao**
Fix namespace gaps for `make mandiff`:
```
Object Namespace::istio-control is missing in A:
Object Namespace::istio-telemetry is missing in A:
Object Namespace::istio-policy is missing in A:
```

### CODE REVIEW

**Comment by ayj** [comment]
I might be running the wrong set of commands, but it looks like the gaps are still present?

```bash
$ curl -L https://github.com/istio/operator/pull/170.patch | patch -up1
$ make mandiff
$ grep -rn "Object Namespace::istio" /tmp/istio-mandiff-out/
/tmp/istio-mandiff-out/mandiff-default-profile.diff:2082:Object Namespace::istio-control is missing in A:
/tmp/istio-mandiff-out/mandiff-default-profile.diff:2138:Object Namespace::istio-system is missing in A:
/tmp/istio-mandiff-out/mandiff-default-profile.diff:2142:Object Namespace::istio-security is missing in A:
/tmp/istio-mandiff-out/mandiff-default-profile.diff:2154:Object Namespace::istio-policy is missing in A:
/tmp/istio-mandiff-out/mandiff-default-profile.diff:2158:Object Namespace::istio-telemetry is missing in A:
```

File: tests/profiles/helm/values-istio-default.yaml [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # default profile will use all the settings in global.yaml
+global:
+  istioNamespace: istio-system
```
Comment (ostromart) [inline tests/profiles/helm/values-istio-default.yaml]: This should not be required, it's translated from IstioControlPlane API.

Comment (morvencao) [reply tests/profiles/helm/values-istio-default.yaml]: This is values file for `hem template ...`

**Comment by morvencao** [comment]
@ayj Could you also patch another PR: https://github.com/istio/operator/pull/171
Here are my results in my env:
```
# curl -L https://github.com/istio/operator/pull/170.patch | patch -up1
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   140    0   140    0     0    151      0 --:--:-- --:--:-- --:--:--   151
100  3450    0  3450    0     0   1994      0 --:--:--  0:00:01 --:--:--  8023
patching file tests/profiles/helm/values-istio-default.yaml
patching file tests/profiles/helm/values-istio-demo-auth.yaml
patching file tests/profiles/helm/values-istio-demo.yaml
patching file tests/profiles/helm/values-istio-minimal.yaml
patching file tests/profiles/helm/values-istio-sds.yaml
# curl -L https://github.com/istio/operator/pull/171.patch | patch -up1
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   140    0   140    0     0    214      0 --:--:-- --:--:-- --:--:--   214
100  830k    0  830k    0     0   375k      0 --:--:--  0:00:02 --:--:--  641k
patching file data/profiles/default.yaml
patching file data/profiles/demo-auth.yaml
patching file data/profiles/demo.yaml
patching file data/profiles/minimal.yaml
patching file data/profiles/sds.yaml
patching file pkg/vfsgen/vfsgen_data.go
# make mandiff
...
# grep -rn "Object Namespace::istio" /tmp/istio-mandiff-out/
/tmp/istio-mandiff-out/mandiff-default-profile.diff:1030:Object Namespace::istio-security is missing in A:
/tmp/istio-mandiff-out/mandiff-demoauth-profile.diff:1755:Object Namespace::istio-security is missing in A:
/tmp/istio-mandiff-out/mandiff-minimal-profile.diff:331:Object Namespace::istio-security is missing in A:
/tmp/istio-mandiff-out/mandiff-sds-profile.diff:1394:Object Namespace::istio-security is missing in A:
```

For the `istio-security` namespace, it's still under discussion at https://github.com/istio/istio/issues/16132

**Comment by morvencao** [comment]
Move istio security components to `istio-system` ns to resolve: https://github.com/istio/istio/issues/16132

**Comment by ostromart** [comment]
@morvencao can this PR be closed, as it appears to be superseded by https://github.com/istio/operator/pull/171 ?

**Comment by morvencao** [comment]
@ostromart Just rebased the changes, I think we still need this PR, it's complementarity the https://github.com/istio/operator/pull/171, as you can see, we still need to updating the security ns from `istio-security` to `istio-system`.

**Pull request merged**